### PR TITLE
noarch

### PR DESCRIPTION
**Pull request by jschueller**
@conda-forge-admin, please rerender

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,8 @@ source:
   sha256: eb2e116e75ecef9d4d228fdc66af54269afa26ab4463042e33785b887c628ba8
 
 build:
-  number: 0
+  number: 1
+  noarch: python
   script: python setup.py install
```
Comment (takluyver) [inline recipe/meta.yaml]: While we're at it, let's switch this to `pip install --no-deps .` , which we're using for most Jupyter repos. The advantage of that is that pip 'sees' the package when it's installed by conda, so it's less likely to try to install it again and cause problems.

**Comment by jschueller** [comment]
ok, I used the recommended setuptools options

**Comment by takluyver** [comment]
Fair enough, I think that has the same effect as using pip.

**Comment by takluyver** [comment]
Ah, no, we didn't change this package to always use setuptools. You will need to use `pip install --no-deps .` instead.

**Comment by takluyver** [comment]
Why did you close this? I think it was fine apart from the need to use `pip` in the build. :-S

**Pull request closed without merging**